src/bin/vip-import-media.js

```diff
@@ -106,17 +106,30 @@ Are you sure you want to import the contents of the URL?
 		'Check the status of a currently running media import or retrieve an error log of the most recent media import.'
 	)
 	.command( 'abort', 'Abort the media import currently in progress.' )
-	.option( 'exportFileErrorsToJson', 'Format the error log in JSON. Default is TXT.' )
+	.option( 'exportFileErrorsToJson', 'Alias for --export-file-errors-to-json. (deprecated)' )
+	.option( 'export-file-errors-to-json', 'Format the error log in JSON. Default is TXT.' )
+	.option( 'saveErrorLog', 'Alias for --save-error-log. (deprecated)' )
 	.option(
-		'saveErrorLog',
+		'save-error-log',
 		'Skip the confirmation prompt and download an error log for the import automatically.'
 	)
+	.option( 'overwriteExistingFiles', 'Alias for --overwrite-existing-files. (deprecated)', false )
 	.option(
-		'overwriteExistingFiles',
+		'overwrite-existing-files',
 		'Overwrite existing files with the imported files if they have the same path and file name.',
 		false
 	)
-	.option( 'importIntermediateImages', 'Include intermediate image files in the import.', false )
+	.option(
+		[ 't', 'allow-all-file-types' ],
```

Comment:
Why `-t`?

Also, since none of the other params have short variants, we should probably skip it for this one as well.

Reply:
By default each param has a short variant (_which is not documented_) and that's the first character of the param. For `allow-all-file-types` it was setting to `-A`. In order to avoid confusion with the application parameter which has `-a` as the short hand, I decided to have separate param for it which is not used for this command.

<img width="1496" alt="Screenshot 2025-02-24 at 8 06 47 PM" src="https://github.com/user-attachments/assets/6200f05f-aec7-46ff-adfb-ed1a4d429fd9" />